term/src/terminalstate.rs

```diff
@@ -2961,17 +2965,45 @@ impl<'a> Performer<'a> {
         for g in unicode_segmentation::UnicodeSegmentation::graphemes(p.as_str(), true) {
             let g = if self.dec_line_drawing_mode {
                 match g {
+                    "`" => "◆",
+                    "a" => "▒",
+
+                    // AZL: I do not know why the next four Unicode glyphs are
```

Comment:
Is that a local font problem?  If I copy and paste the characters from your PR description into the terminal I see those glyphs in my terminal on this mac.  If you do the same, do you see those glyphs on your system, or is the issue that the don't show up when used in the appropriate mode when rendered via the terminal?

wezterm will log information about glyphs that it failed to resolve.  It can take several seconds to do that if you're in a debug build, because it has to parse and compute glyph coverage maps for a potentially large list of fallback fonts, and that's really slow in a debug build.

Reply:
I saw nothing logged to stderr on the glyphs.  wezterm ls-fonts yields:

```
 Primary font:
wezterm.font_with_fallback({
  -- <built-in>, BuiltIn
  "JetBrains Mono",

  -- /usr/share/fonts/truetype/noto/NotoColorEmoji.ttf, FontConfig
  "Noto Color Emoji",

  -- <built-in>, BuiltIn
  "Last Resort High-Efficiency",

})
```

The other fonts are all the same: JetBrains Mono, Noto, Last Resort.

When I paste " ␉ ␌ ␍ ␊ ␋" into cat I see these instead:

![glyphs](https://user-images.githubusercontent.com/4357501/122657236-626a5e80-d127-11eb-89da-0e200edce6fa.png)

I've got the format fixed, and will add the test.  (This is fun! :) )

Reply:
I think the glyphs you're seeing are coming from a fallback font; they seem like alternative pictorial versions of `␉ ␌ ␍ ␊ ␋`.   I think I should add a thing to `ls-fonts` to have it explain which fonts are used for a given text input to help understand cases like this!


> I've got the format fixed, and will add the test. (This is fun! :) )

Great!

Reply:
`wezterm ls-fonts --text "␉ ␌ ␍ ␊ ␋"` will explain which fonts were used now in `main`